recipe/meta.yaml

```diff
@@ -34,12 +24,12 @@ requirements:
       - {{ compiler('c') }}
       - {{ compiler('cxx') }}
       - pkg-config
-      - python 2.*
+      - python >=2.7,<3
       - mako
       - zlib
   host:
       - pthread-stubs
-      - llvmdev
+      - llvmdev <8.0.0
```

Comment:
This may not be necessary anymore but this puts the version of `llvmdev` around the time of the release of this version of `mesalib`.  I'll relax this when we update to a newer `mesalib` release.